vector/src/main/res/values/strings.xml

```diff
@@ -2566,8 +2572,20 @@
     <string name="auth_pin_reset_title">Reset pin</string>
     <string name="auth_pin_new_pin_action">New pin</string>
     <string name="auth_pin_reset_content">To reset your PIN, you\'ll need to re-login and create a new one.</string>
+    <string name="settings_security_application_protection_title">Protect access to the application.</string>
+    <string name="settings_security_application_protection_summary">Protect access to the application using pin code and biometrics.</string>
+    <string name="settings_security_application_protection_screen_title">Configure protection</string>
     <string name="settings_security_pin_code_title">Enable PIN</string>
     <string name="settings_security_pin_code_summary">If you want to reset your PIN, tap Forgot PIN to logout and reset.</string>
+    <string name="settings_security_pin_code_use_biometrics_title">Use biometrics</string>
+    <string name="settings_security_pin_code_use_biometrics_summary_on">Depending on your device, you will be able to unlock using fingerprints, face recognition, iris recognition, etc.</string>
+    <string name="settings_security_pin_code_use_biometrics_summary_off">PIN code is the only way to unlock the application.</string>
+    <string name="settings_security_pin_code_notifications_title">Display complete notifications</string>
```

Comment:
**Suggestion 1**
Show notifications content on lock screen
_More specific on where notifications will appear, and what we mean by "complete"_

Reply:
@niquewoodhouse this is not only on lock screen. _Complete_ means with full details.

Reply:
@bmarty thanks for explaining :) My only suggestion left is "Show content in notifications" or "Show notification contents" because I think its clearer to a user that this means the content of a notification.

Reply:
Ok, that's far better from my wording, thanks!